aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -753,6 +754,31 @@ func resourceAviatrixTransitGateway() *schema.Resource {
 					},
 				},
 			},
+			"interface_mapping": {
```

Comment:
do we still need this after your recent changes ?

Reply:
Hi @ssundarrajan, yes we need interface mapping for only the esxi devices. By default if the user provides no value for this then we will be setting the interface mapping to dell configuration. This was implemented after my discussion with Purnima and Rui.